src/IO/WriteHelpers.cpp

```diff
@@ -34,6 +36,79 @@ void formatUUID(std::reverse_iterator<const UInt8 *> src16, UInt8 * dst36)
     formatHex(src16 + 2, &dst36[24], 6);
 }
 
+/** Further we want to generate constexpr array of strings with sizes from sequence of unsigned ints [0..N)
+ *  in order to use this arrey for fast conversion of unsigned integers to strings
+ */
```

Comment:
There is the same predefined [table] in (https://github.com/ClickHouse/clickhouse-private/blob/2b3cb810cd791a7409d5bc27878fcccb71ad5d69/src/Common/formatIPv6.cpp#L13) `formatIPv6.cpp`. I think it's better to leave only one implementation. But I'm not sure which one is better: explicit or generated one.